xml/System.Security.Cryptography.Xml/SignedXml.xml

```diff
@@ -65,16 +65,16 @@
   
  The main parts of this structure are:  
   
- The <`CanonicalizationMethod`> element  
```

Comment:
This is completely unrelated to your PR, but it would be nice to turn these six elements into a bulleted list.

Reply:
Thanks @rpetrusha. I've implemented the changes + some others I noticed that would be beneficial for our markdig migration.